packages/devtools-core/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.ui.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "lib",
+    "target": "es2017",
+    "module": "esnext"
+  },
+  "include": ["src", "typings"],
+  "references": [{ "path": "../core" }, { "path": "../logger-plugin" }]
```

Comment:
I don't think this depends on the `logger-plugin`

Reply:
Oops, that is correct